packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/ports/ports.constants.js

```diff
@@ -0,0 +1,13 @@
+export const PACK_XDSL = {
```

Comment:
Maybe we should rename/split this `PACK_XDSL` constant (too generic):
```js

export const COMMON = {
  min: 1,
  max: 65535,
  maxlength: 5,
};

export const INTERNAL = {
  startPlaceholder: '8080',
  endPlaceholder: '8085',
};

export const EXTERNAL = {
  startPlaceholder: '80',
  endPlaceholder: '85',
};

Reply:
OK. I note this for a next feature.